src/azure-cli/azure/cli/command_modules/network/zone_file/parse_zone_file.py

```diff
@@ -207,6 +208,8 @@ def _serialize(tokens):
     for tok in tokens:
         if tok is None:
             continue
+        elif tok == '':
+            tok = 'EMPTY'
```

Comment:
can this "EMPTY" affect other records?

Reply:
I don't think we allow empty value in any other field for any other record type. Otherwise, I guess serialize function call after this tokenize function call will remove that entirely.

Reply:
We don't allow empty string for any other record type from azcli. This should be good enough.